NDB.Covid19/NDB.Covid19/ViewModels/ENDeveloperToolsViewModel.cs

```diff
@@ -245,32 +246,52 @@ public string GetExposureInfosFromLastPull()
         // Consider: Displaying the exposure configuration in the activities.
         public async Task<string> FetchExposureConfigurationAsync()
         {
-            EN.Configuration c = await new ExposureNotificationHandler().GetConfigurationAsync();
-            string res = $" AttenuationWeight: {c.AttenuationWeight}, Values: {EnConfArrayString(c.AttenuationScores)} \n" +
-                $" DaysSinceLastExposureWeight: {c.DaysSinceLastExposureWeight}, Values: {EnConfArrayString(c.DaysSinceLastExposureScores)} \n" +
-                $" DurationWeight: {c.DurationWeight}, Values: {EnConfArrayString(c.DurationScores)} \n" +
-                $" TransmissionWeight: {c.TransmissionWeight}, Values: {EnConfArrayString(c.TransmissionRiskScores)} \n" +
-                $" MinimumRiskScore: {c.MinimumRiskScore}" +
-                $" DurationAtAttenuationThresholds: [{c.DurationAtAttenuationThresholds[0]},{c.DurationAtAttenuationThresholds[1]}]";
-
-            Debug.WriteLine("Exposure Configuration:");
-            Debug.WriteLine($" AttenuationWeight: {c.AttenuationWeight}, Values: {EnConfArrayString(c.AttenuationScores)}");
-            Debug.WriteLine($" DaysSinceLastExposureWeight: {c.DaysSinceLastExposureWeight}, Values: {EnConfArrayString(c.DaysSinceLastExposureScores)}");
-            Debug.WriteLine($" DurationWeight: {c.DurationWeight}, Values: {EnConfArrayString(c.DurationScores)}");
-            Debug.WriteLine($" TransmissionWeight: {c.TransmissionWeight}, Values: {EnConfArrayString(c.TransmissionRiskScores)}");
-            Debug.WriteLine($" MinimumRiskScore: {c.MinimumRiskScore}");
-
-            DevToolsOutput = res;
+            try
+            {
+                DailySummaryConfiguration dsc = await new ExposureNotificationHandler().GetDailySummaryConfigurationAsync();
+                string result = $"AttenuationThresholds: {EnConfArrayString(dsc.AttenuationThresholds)}\n " +
+                    $"AttenuationWeights: {EnConfDictionaryString(dsc.AttenuationWeights)} \n" +
+                    $"DaysSinceLastExposureThreshold: {dsc.DaysSinceLastExposureThreshold} \n" +
+                    $"DaysSinceOnsetInfectiousness: {EnConfDictionaryString(dsc.DaysSinceOnsetInfectiousness)} \n" +
+                    $"DefaultInfectiousness: {dsc.DefaultInfectiousness}\n" +
+                    $"DefaultReportType: {dsc.DefaultReportType}\n" +
+                    $"InfectiousnessWeights: {EnConfDictionaryString(dsc.InfectiousnessWeights)}\n" +
+                    $"ReportTypeWeights: {EnConfDictionaryString(dsc.ReportTypeWeights)}\n";
+                DevToolsOutput = result;
+            }
+            catch (Exception)
```

Comment:
What exception is caught here?

Reply:
Looks like it's InvalidOperationException. Thrown by the EN Xamarin library if v2 is not available. Fixed now.

Reply:
Are you sure no other exceptions are thrown? Cause I guess we want the following:
1. try V2
2. if illegal is thrown - try v1
3. if something else is thrown - abort/log/display error

Reply:
It might throw whatever we implement in the handler method, but I don't think anything else